cpp/test/sg/rf_accuracy_test.cu

```diff
@@ -81,27 +81,29 @@ class RFClassifierAccuracyTest : public ::testing::TestWithParam<RFInputs> {
 
  private:
   void setRFParams() {
-    DecisionTree::DecisionTreeParams tree_params;
+    //DecisionTree::DecisionTreeParams tree_params;
```

Comment:
Can we remove this line instead of commenting it?

Reply:
yep, done